library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java

```diff
@@ -27,9 +29,10 @@
     String SCREEN_PRIVACY_POLICY = "Privacy Policy";
     String SCREEN_COPYRIGHT = "Copyright Info";
 
-    /* Custom dimensions */
-    int DIMENSION_TOOL = 1;
-    int DIMENSION_LANGUAGE = 2;
+    /* Action event names */
+    String ACTION_SHARE = "Share Icon Engaged";
```

Comment:
Is there any reason to not make these and the ones above `static final` Strings?

Reply:
I wondered about that right after I posted the comment. It's been awhile since I've done java :)